subworkflows/preprocess_rnaseq_bam.cwl

```diff
@@ -1,7 +1,8 @@
 cwlVersion: v1.0
 class: Workflow
 doc: >-
-    "Sambamba Sort Mark Dup Sub WF"
+    "Preprocessing sub workflow for GATK4 RNAseq SNV calling.
+    Uses Sambamba for mark dups and sort, GATK SplitNCigarReads for final processed bam output"
```

Comment:
While sambamba is much more efficient than Picard, the implementations are not exactly the same. I'm not so concerned about it, but it's important to be aware of this.

Reply:
Thanks! I've made more updates in which I use sambamba just once - no intermediate sort steps. As for picard compatibility, I can do some benchmarking, but am also ok with general consensus that the effect is minimal.

Reply:
Added a qualifying sentence to the readme. A fair amount of updates from the last PR...I could try and close this one and instead do it by release tag to that PR #1 goes to v0.3, PR # v0.4